### PR TITLE
#1404 Corrected Repeated contact-us

### PR DESCRIPTION
**Pull request by PVBharadwaj**
### 🛠️ Fixes Issue

#1404 

### 👨‍💻 Changes proposed

In the resources section contact us appeared two times. It is fixed.

### ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### 📷 Screenshots
The webpage was previously like the screenshot shown below 
Screenshot 2023-07-01 160045

It was changed to this.
image

### CODE REVIEW

**Comment by PVBharadwaj** [comment]
@rohanbabbar04  please review this pr under GSSoC'23

File: resource.html [hunk 1]
```diff
@@ -1351,26 +1339,23 @@ <h2 class="hover-underline-animation">Regional Language Book Section</h2>
                 </div>
             </div>
             <div class="lower">
-                <div class="topic"><a href="./public/contact us .html" style="color: antiquewhite;">Contact Us
-                    </a></div>
+                <!-- <div class="topic"><a href="./public/contact us .html" style="color: antiquewhite;">Contact Us
+                    </a></div> -->
```
Comment (rohanbabbar04) [inline resource.html:1343]: Remove this line, don't comment code...

**Review by rohanbabbar04 — CHANGES_REQUESTED** [review]
image
Make sure it works for all screen sizes....

**Comment by PVBharadwaj** [comment]
@rohanbabbar04  I have made the required changes and created a new pr ( #1411 ) for the same issues. Please close this pr and review that pr

**Pull request closed without merging**